android/app/google-services.enc.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
If I were to change the google-services.json file. What are the steps needed and can we have this document separately under ./docs directory?

Reply:
I'm currently creating documentation on confluence which explains the process here.